_examples/docscansandbox/demo_test.go

```diff
@@ -0,0 +1,338 @@
+		Build()
+	if err != nil {
+		return err
+	}
+
+	textDataCheckBreakdown, err := report.NewBreakdownBuilder().
+		WithSubCheck("document_in_date").
+		WithResult("PASS").
+		Build()
+	if err != nil {
+		return err
+	}
+
+	textDataCheckRecommendation, err := report.NewRecommendationBuilder().
+		WithValue("APPROVE").
+		Build()
+
+	textDataCheck, err := check.NewDocumentTextDataCheckBuilder().
+		WithBreakdown(textDataCheckBreakdown).
+		WithRecommendation(textDataCheckRecommendation).
+		WithDocumentFields(map[string]string{
+			"full_name":       "John Doe",
+			"nationality":     "GBR",
+			"date_of_birth":   "1986-06-01",
+			"document_number": "123456789",
+		}).
+		WithDocumentFilter(documentFilter).
+		Build()
+	if err != nil {
+		return err
+	}
+
+	checkReports, err := request.NewCheckReportsBuilder().
+		WithAsyncReportDelay(5).
+		WithDocumentAuthenticityCheck(documentAuthenticityCheck).
+		WithDocumentTextDataCheck(textDataCheck).
+		Build()
+	if err != nil {
+		return err
+	}
+
+	textExtractionTask, err := task.NewDocumentTextDataExtractionTaskBuilder().
+		WithDocumentFields(map[string]string{
+			"full_name":       "John Doe",
+			"nationality":     "GBR",
+			"date_of_birth":   "1986-06-01",
+			"document_number": "123456789",
+		}).
+		Build()
+	if err != nil {
+		return err
+	}
+
+	taskResults, err := request.NewTaskResultsBuilder().
+		WithDocumentTextDataExtractionTask(textExtractionTask).
+		Build()
+	if err != nil {
+		return err
+	}
+
+	responseConfig, err := request.NewResponseConfigBuilder().
+		WithCheckReports(checkReports).
+		WithTaskResults(taskResults).
+		Build()
+	if err != nil {
+		return err
+	}
+
+	configErr := c.client.ConfigureSessionResponse(c.getIFrameSessionID(), responseConfig)
+
+	if configErr != nil {
+		request, _ := json.Marshal(responseConfig)
+		return errors.New(string(request) + configErr.Error())
+	}
+
+	return configErr
+}
+
+func (c *webContext) iAmOn(path string) error {
+	return c.wd.Get(fmt.Sprintf("%s%s", appBaseUrl, path))
+}
+
+func (c *webContext) clickOn(selector string) error {
+	elem, err := c.wd.FindElement(selenium.ByCSSSelector, selector)
+	if err != nil {
+		panic(err)
+	}
+
+	return elem.Click()
+}
+
+func (c *webContext) iChoose(value string) error {
+	return c.clickOn(fmt.Sprintf("input[value='%s']", value))
+}
+
+func (c *webContext) iClickOnAddPhotoButton() error {
+	return c.clickOn(fmt.Sprintf("*[data-qa='addPhotoButton']"))
+}
+
+func (c *webContext) iClickOnFinishButton() error {
+	return c.clickOn(fmt.Sprintf("*[data-qa='finish-button']"))
+}
+
+func (c *webContext) iShouldSee(text string) error {
+	source, err := c.wd.PageSource()
+	if err != nil {
+		return err
+	}
+	if !strings.Contains(source, text) {
+		return fmt.Errorf("Page source does not contain \"%s\"", text)
+	}
+	return nil
+}
+
+func (c *webContext) iSwitchToTheIframe() error {
+	iFrame, err := c.wd.FindElement(selenium.ByTagName, "iframe")
+	if err != nil {
+		panic(err)
```

Comment:
Looks like we're returning the error in most functions in this class, would it be better to return this error too instead of panicking?
```suggestion
		return err
```

Reply:
Yes, makes sense to return the error instead